cmd/site-controller/deploy.yaml

```diff
@@ -86,15 +86,16 @@ rules:
   - delete
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
 metadata:
   labels:
     application: skupper-site-controller
   name: skupper-site-controller
 subjects:
 - kind: ServiceAccount
   name: skupper-site-controller
+  namespace: default
```

Comment:
Should probably add an explicit namespace to the serviceaccount definition above if we need to specify one here. As above, if possible, I'd like to keep the ability to run without cluster level rights, so if we had the env var for the site controller as described above, we could then have two separate deploy yamls, one to watch all namespaces, one to watch the current namespace.

Reply:
Do you think it would be helpful to have two deploy .yamls? One to watch "all" and one to watch current namespace?

Reply:
I think it would be great if we could, yes. It may end up that no one needs the version that watches the current namespace, but it looks like a relatively small effort is required to keep that option open until we know for sure.

Reply:
Nice work on making this configurable, thanks!